pkg/lib/configreader/reader.go

```diff
@@ -891,7 +891,7 @@ func ParseYAMLFile(dest interface{}, validation *StructValidation, filePath stri
 
 	errs := Struct(dest, fileInterface, validation)
 	if errors.HasError(errs) {
-		return errors.WrapAll(errs, filePath)
+		return errors.WrapAll(errs, files.PathRelativeToCWD(filePath))
```

Comment:
I think I prefer the abolute path to relative paths because I can copy and paste the path easily without worrying about where I am.